packages/web/src/test/test-utils.tsx

```diff
@@ -1,16 +1,16 @@
 import { ReactElement, ReactNode } from 'react'
 
-import { DeepPartial } from '@audius/common/utils'
 import { ThemeProvider } from '@audius/harmony'
 import { render, RenderOptions } from '@testing-library/react'
+import { PartialDeep } from 'type-fest'
 
 import { ReduxProvider } from 'app/ReduxProvider'
 import { RouterContextProvider } from 'components/animated-switch/RouterContextProvider'
 import { ToastContextProvider } from 'components/toast/ToastContext'
 import { AppState } from 'store/types'
 
 type TestOptions = {
-  reduxState?: DeepPartial<AppState>
+  reduxState?: PartialDeep<AppState>
```

Comment:
is there a difference in these two?

Reply:
no, but one comes from type-fest which i trust more